features/references/manage_working_patterns.feature

```diff
@@ -0,0 +1,34 @@
+# FeatureID  EpicID StoryID TestID
+Feature: Identify Judicial Office Holder Working Patterns and allocate to hearing that they are working and available
+
+  As a Solution
+  I want to be able to identify a judicial office officer's daily and weekly working pattern.
+  So that the hours worked each day and the days of the week worked can be applied to the scheduling and listing of hearings.
+
+  Background:
+    Given Hearing are setup
```

Comment:
as mentioned in previous file reviews, update this statement and add JOH creation step too